src/net_processing.cpp

```diff
@@ -747,27 +747,22 @@ bool ProcessMessage(CNode *pfrom, std::string strCommand, CDataStream &vRecv, in
     // messages during the initial (x)version handshake).
     else if (strCommand == NetMsgType::PING)
     {
-        // take the lock exclusively to force a serialization point
```

Comment:
Is the "force serialization point" still being done, or intentionally removed?

Reply:
i guess i must have lost a line of code in a rebase somewhere. it is supposed to be locking the upgrade section instead of using the CSharedUnlocker. good catch.